lib/ckb/always_success_wallet.rb

```diff
@@ -10,63 +10,62 @@ def initialize(api)
       @api = api
     end
 
+    # @param target_lock [Ckb::Script]
+    # @param capacity [Integer]
     def send_capacity(target_lock, capacity)
       i = gather_inputs(capacity, MIN_CELL_CAPACITY)
       input_capacities = i.capacities
 
       outputs = [
-        {
+        Output.new(
           capacity: capacity.to_s,
-          data: "0x",
           lock: target_lock
-        }
+        )
       ]
+
       if input_capacities > capacity
-        outputs << {
+        outputs << Output.new(
           capacity: (input_capacities - capacity).to_s,
-          data: "0x",
-          lock: lock_script_json_object
-        }
+          lock: lock_script
+        )
       end
-      tx = {
+
+      tx = Transaction.new(
         version: 0,
         deps: [],
         inputs: i.inputs,
-        outputs: outputs,
-        witnesses: []
-      }
+        outputs: outputs
+      )
       api.send_transaction(tx)
     end
 
     def install_mruby_cell!(mruby_cell_filename)
       data = File.read(mruby_cell_filename)
-      cell_hash = Ckb::Utils.bin_to_prefix_hex(Ckb::Blake2b.digest(data))
-      output = {
+      cell_hash = Ckb::Utils.bin_to_hex(Ckb::Blake2b.digest(data))
```

Comment:
Can `install_mruby_cell` install C or other cells other than `mruby`'s? If so, perhaps we should rename this to be more precise.

Reply:
Maybe install `mruby` cell is this method only usage for this demo ..

Reply:
But it doesn't stop us from installing a C cell right?

Reply:
Yes, but I think we should not rename it to `install_cell!`, we don't encourage user to install any other cells, it's only usage is install `mruby` cell for run ruby scripts.

Reply:
Makes sense. Let's keep is as.